theano/gpuarray/extra_ops.py

```diff
@@ -450,12 +450,13 @@ def c_support_code_struct(self, node, nodename):
         """ % locals()
         return super(GpuCumsum, self).c_support_code_struct(node, nodename) + code
 
-
+@register_opt('fast_compile')
 @op_lifter([CumsumOp])
-def use_gpu_cumsumop(node, ctx_name):
-    if node.inputs[0].dtype == 'float32':
-        axis = node.op.axis
-        x = node.inputs[0]
+@register_opt2([CumsumOp], 'fast_compile')
```

Comment:
Okay, I guess I made some errors here. This is not actually an op that used to be an `op_lifter` before. From the diff between master and this branch gives me this, 
```
diff --git a/theano/gpuarray/extra_ops.py b/theano/gpuarray/extra_ops.py
index b2ef45e..24ccf06 100644
--- a/theano/gpuarray/extra_ops.py
+++ b/theano/gpuarray/extra_ops.py
@@ -10,7 +10,7 @@ except ImportError:
 
 from .basic_ops import (as_gpuarray_variable, GpuKernelBase, Kernel,
                         infer_context_name, GpuFromHost)
-from .opt import register_opt, op_lifter, register_opt2
+from .opt import register_opt as register_gpu_opt, op_lifter
 ```
I had grep'ed for op_lifter and hence added the new Opt. I guess I need to pull this change back ?